tests/VolleyManagement.Domain.UnitTests/PlayerTests.cs

```diff
@@ -7,6 +7,38 @@ namespace VolleyManagement.Domain.UnitTests
 {
     public class PlayerTests
     {
+        [Fact]
+        public void Player_ValidFields_ExceptionIsNowThrown()
```

Comment:
_CreateValidInstance_PlayerCreated

And check if everything has been created correctly